packages/next-swc/crates/next-core/src/next_config.rs

```diff
@@ -353,6 +355,7 @@ pub struct ExperimentalConfig {
     pub app_dir: Option<bool>,
     pub server_components_external_packages: Option<Vec<String>>,
     pub turbo: Option<ExperimentalTurboConfig>,
+    pub compiler: Option<CompilerConfig>,
```

Comment:
Is there a `experimental.compiler` option?

Reply:
no, this is my fault.